etc/DependencyInstaller.sh

```diff
@@ -309,6 +317,14 @@ while [ "$#" -gt 0 ]; do
     shift 1
 done
 
+if [[ "${option}" == "none"  ]]; then
+        echo "You must use one of: -all|-base|-common" >&2
+        _help
```

Comment:
Very good. 

Add "exit 1" here, no argument is an error

Reply:
`_help` already does `exit 1`. Would you like for me to add it as a redundancy?

Reply:
Ah! Good! Much better.

Ignore my "add exit 1" comment here.